projects/batfish-client/src/org/batfish/client/Client.java

```diff
@@ -315,6 +325,176 @@ private boolean answer(String questionTemplateName, String paramsLine,
       return execute(wItemAs, outWriter);
    }
 
+   /**
+    * Validate the contents contained in json-encoded {@code value}
+    * matches the type required by {@code variable}.
+    * Call {@link Variable#getType()} on {@code variable} shows the expected
```

Comment:
Why newline before `Call`? If it's a new paragraph, add a blank line and a `<p>`. If it's part of the same paragraph, merge into above.